ontology.ttl

```diff
@@ -160,7 +160,28 @@ hobbit:organizer
 		rdfs:label "Organizer"^^xsd:string ;
 		rdfs:domain hobbit:Challenge ;
 		rdfs:range xsd:string .
-				
+
+hobbit:registrationCutoffDate
+		rdf:type owl:DatatypeProperty ;
+		rdfs:comment "Date after which no more changes to the systems participating can be made. No more experiments will run after this date."^^xsd:string ;
```

Comment:
That is not fully correct. I would rephrase it to
`Date after which no more systems can be registered for a challenge.`
Experiments will run after this date (i.e., the last "wave" of experiments) and as long as the experiment of a certain system hasn't been executed, its images still can be updated.